lib/arena.rb

```diff
@@ -1,2 +1,65 @@
+require "pry"
+
 class Arena
-end
+  attr_accessor(:name, :gladiators, :vote, :losing_gladiators)
+
+  def initialize(name)
+      @name = name.capitalize
+      @gladiators = []
+  end
+
+  def add_gladiator(gladiator)
+    return if @gladiators.length >= 2
+    @gladiators << gladiator
+  end
+
+  def remove_gladiator_name
+    @gladiators.reject! { |gladiator| gladiator.name == "Hercules"} #.name refers to attribute of @gladiators, "gladiator" can say anything, it's like "i".
```

Comment:
Why are we always removing Hercules? If we pass in a gladiator name, we could remove the named gladiator.
```
def remove_gladiator(name)
     gladiators.reject! { |gladiator| gladiator.name == name}
end
```